examples/multimodal/multimodal_llm/neva/conf/neva_config.yaml

```diff
@@ -186,12 +187,22 @@ model:
     num_workers: 8
     dataloader_type: cyclic
     data_path:
+      # could be a path to a single file or a list of files for data blending like below
```

Comment:
add warning here in comments and later.
If use blendable dataset the epoch won't make too much sense anymore, depends on your dataset size and weights it might under sample some dataset.

Reply:
added more comments to address data blending